src/engine/renderer/tr_shader.cpp

```diff
@@ -1771,7 +1840,7 @@ void LoadExtraMaps( shaderStage_t *stage, const char *colorMapName )
 					diffuseMap  textures/map_solarium/water4/water4
 					normalMap   textures/map_solarium/water4/water4_norm
 					specularMap textures/map_solarium/water4/water4_gloss
-					normalScale 1 -1 1
+					normalFormat + + +
```

Comment:
Is this right? It looks like the letters are required.

Reply:
hmm, yes will fix the comment